applications/matter_bridge/README.rst

```diff
@@ -363,24 +359,22 @@ After building the sample and programming it to your development kit, complete t
             .. parsed-literal::
                :class: highlight
 
-               uart:~$ matter_bridge add *<device_type>* *<bluetooth_device_index>*
+               uart:~$ matter_bridge add *<bluetooth_device_index>*
 
-            The *<device_type>* is a Matter device type that will be used to represent a new bridged device in the Matter Data Model.
-            See the description in :ref:`matter_bridge_cli` for the list of supported values.
             The *<bluetooth_device_index>* is a Bluetooth LE device index that was scanned in the previous step.
-            For example, if you want to add a new Bluetooth LE bridged device with index `0` that is exposed as an On/Off Light endpoint, use the following command:
+            For example, if you want to add a new Bluetooth LE bridged device with index `1`, use the following command:
```

Comment:
```suggestion
            For example, if you want to add a new Bluetooth LE bridged device with index ``1``, use the following command:
```
Just a formatting difference since we are talking about a specific value that is going to be input.